docs/feature_flags.md

```diff
@@ -203,6 +205,10 @@ When using offset with duration expressions, you must wrap the expression in
 parentheses. Without parentheses, only the first duration value will be used in
 the offset calculation.
 
+Additionally, `step()` can be used in duration expressions.
```

Comment:
PR is also adding `min` and `max` operators, let's mention them here if it was intentional.